azure-pipelines.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 

```

Comment:
Below we install ODBC 17.3 - should this be updated?

Reply:
Did you mean Windows? Not for now because the job is only a placeholder for future development (see the condition is `false`)